core/modules/user/user.module

```diff
@@ -2263,21 +2253,30 @@ function user_roles($members_only = FALSE, $permission = NULL, $full_objects = F
 }
 
 /**
- * Helper for user_roles().
+ * Checks if access should be restricted.
```

Comment:
A more verbose description should be used here. What makes a role "restricted"? The docblock should describe it without requiring the developer to read the source code.